src/js/utils/dom.js

```diff
@@ -86,6 +86,23 @@ export function isEl(value) {
 }
 
 /**
+ * Determines if the player is embedded in an iframe.
+ *
+ * @return {boolean}
+ *
+ */
+export function isPlayerInFrame() {
```

Comment:
I propose changing this to `isInIframe` more generally. Might be pedantic or whatever, but that seems like a better name.